wordle-web/plugins/vuetify.ts

```diff
@@ -1,42 +1,100 @@
 // import this after install `@mdi/font` package
-import "@mdi/font/css/materialdesignicons.css";
+import '@mdi/font/css/materialdesignicons.css'
 
-import "vuetify/styles";
-import { createVuetify } from "vuetify";
+import 'vuetify/styles'
+import { createVuetify } from 'vuetify'
 
 export default defineNuxtPlugin((app) => {
   const vuetify = createVuetify({
     theme: {
-      defaultTheme: "dark",
+      defaultTheme: 'dark',
       themes: {
         light: {
           colors: {
-            primary: "#673AB7",
-            secondary: "#FF9800",
-            error: "#FF5252",
-            info: "#2196F3",
+            primary: "#C2185B",
+            secondary: "#8E24AA",
+            error: "#DD2C00",
+            info: "#64B5F6",
             success: "#4CAF50",
-            warning: "#FFC107",
-            correct:"#4CAF50",
-            misplaced:"#FFC107",
-            wrong:"#BDBDBD"
+            correct: "#4CAF50",
+            misplaced: "#FFFF00",
+            wrong: "#BDBDBD",
+            unknown: "#d6d6d6"
           },
         },
         dark: {
           colors: {
-            primary: "#673AB7",
-            secondary: "#FF9800",
-            error: "#FF5252",
-            info: "#2196F3",
+            primary: "#C2185B",
+            secondary: "#8E24AA",
+            error: "#DD2C00",
+            info: "#64B5F6",
             success: "#4CAF50",
-            warning: "#FFC107",
-            correct:"#4CAF50",
-            misplaced:"#FFC107",
-            wrong:"#616161"
-          },
+            correct: "#4CAF50",
+            misplaced: "#FFFF00",
+            wrong: "#757575",
+            unknown: "#424242"
+          }
+        },
+        sansLight: {
+          dark: false,
+          colors:{
+            primary: "#0071E3",
+            secondary: "#FF00FF",
+            error: "#FF0000",
+            info: "#64B5F6",
+            success: "#00E5E5",
+            correct: "#00E5E5",
+            misplaced: "#FEFE00",
+            wrong: "#424242",
+            unknown: "#757575"
+          }
+        },
+          sansDark:{
```

Comment:
Didn't think to do this light/dark theme for all the settings. Nice implementation.

Reply:
thank you